packaging/cmake/Config.cmake.in

```diff
@@ -0,0 +1,20 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+if(@WITH_ZLIB@)
```

Comment:
I dont know CMake so I don't really know what any of this means.
If pkg-config is available it should be used to look up compiler and linker-flags for dependency libraries.